src/main/java/com/dumbdogdiner/stickyapi/common/util/url/URLUtil.java

```diff
@@ -44,18 +46,30 @@ public static URLPair findURL(@NotNull String text) {
      *             formatted URLs.
      * @return {@link TextComponent}
      */
-    public static TextComponent convertURLs(@NotNull String text) {
+
+    public static TextComponent convertURLs(String text) {
```

Comment:
Please use param annotations!

Also, this is unrelated to this PR, however I personally don't mind as long as you list it somewhere.